src/Modules/TasksModule/components/TasksList/TasksList.css

```diff
@@ -0,0 +1,31 @@
+:root{
```

Comment:
We need to make this file a css modules to avoid any classes conflicts with any global classes.